action.yml

```diff
@@ -69,6 +76,13 @@ runs:
 
         # Optional (defaults to the current repository).
         # repository: "owner/repo"
+    - uses: crazy-max/ghaction-import-gpg@v3
+      if: ${{ inputs.sign_commits == 'true' }}
```

Comment:
Can we replace `inputs.sign_commits == 'true'` with a check on `gpg_private_key` being empty?